src/components/UserInfo.js

```diff
@@ -1,38 +1,58 @@
-/**
- * "The UserInfo class is responsible for rendering information about the user on the page."
- */
 export default class UserInfo{
   #nameElement;
   #descriptionElement;
+  #avatarElement;
   #name;
   #description;
+  #avatar;
+
+/**
+ * Manage getting and setting information in profile section of page
+ * @param {object} elements {nameElement, descriptionElement, avatarElement} elements containing relevant profile information
+ * @param {object} userInfo {name, about, avatar} strings of user information to popuplate profile information on page
+ */
+  constructor(elements, userInfo){
+    //elements for name, description, and avatar
+    this.#nameElement = elements.nameElement;
+    this.#descriptionElement = elements.descriptionElement;
+    this.#avatarElement = elements.avatarElement;
+
+    //store name, description, and avatar in object
+    this.#name = userInfo.name;
+    this.#description = userInfo.about;
+    this.#avatar = userInfo.avatar;
+
+    //display name, description, and avatar in page
```

Comment:
Needs correction: You should set the contents via `  setUserInfo` and `setAvatar` methods when initializing the user